.gitignore

```diff
@@ -26,3 +26,8 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 debug.log
+
+# firebase
```

Comment:
Did you add these manually? I think we actually need these and they don't expose anything sensitive

Reply:
Re-added them back after convo.